core/src/test/scala/org/apache/spark/deploy/rest/StandaloneRestSubmitSuite.scala

```diff
@@ -676,6 +710,7 @@ private class FaultyStandaloneRestServer(
   protected override val killAllRequestServlet = new InvalidKillAllServlet
   protected override val statusRequestServlet = new ExplodingStatusServlet
   protected override val clearRequestServlet = new ExplodingClearServlet
+  protected override val readyzRequestServlet = new ExplodingReadyzServlet
```

Comment:
nit: Class doc of `FaultyStandaloneRestServer` is not updated.

Reply:
Sure!